lnwire/single_funding_request.go

```diff
@@ -68,13 +68,19 @@ type SingleFundingRequest struct {
 	// supported: P2PKH, P2WKH, P2SH, and P2WSH.
 	DeliveryPkScript PkScript
 
+	// OurDustLimit is the threshold below which no HTLC output should be
+	// generated for our commitment transaction; ie. HTLCs below
+	// this amount are not enforceable onchain from our point view.
+	OurDustLimit btcutil.Amount
```

Comment:
Minor nit: can you drop the `Our|Their` prefix from the `DustLimit` fields here? The PoV context really doesn't apply here since these are wire messages rather than state which is stored locally.